src/addons/model.js

```diff
@@ -1,19 +1,20 @@
 /**
  * @fileoverview A general modelling class used by the Presenter's
  * getModel function.
+ * @flow
  */
 
-import { set, Emitter } from '../microcosm'
+import { type Microcosm, Emitter, set } from '../microcosm'
 
-function isObservable(binding) {
+function isObservable(binding: *): boolean {
   return binding && typeof binding.subscribe === 'function'
 }
 
-function isCallable(binding) {
+function isCallable(binding: *): boolean {
```

Comment:
but doesn't leaving it out let flow infer it also? I don't know what the difference is here?